.travis.yml

```diff
@@ -1,20 +1,26 @@
-sudo: false
 language: php
 
-php:
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7.0
-  - 7.1
-  - 7.2
-  - hhvm
+matrix:
+  include:
+    - php: 5.4
+      dist: trusty
+    - php: 5.5
+      dist: trusty
+    - php: 5.6
+      dist: xenial
+    - php: 7.0
+      dist: xenial
+    - php: 7.1
+      dist: bionic
+    - php: 7.2
+      dist: bionic
+    - php: 7.3
+      dist: bionic
+    - php: 7.4
+      dist: bionic
+    - php: hhvm-3.24
```

Comment:
Just wondering, why this particular version?

Reply:
Because it's LTS, and the tests failed on all lower versions of HHVM LTS.

Reply:
And composer and PHPUnit don't work work HHVM 4+.